### PR TITLE
User: Add users with authentication

### PR DESCRIPTION
**Pull request by nicordev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-1. Run `php bin/console doctrine:migrations:migrate` to create the right tables
+# BileMo
+
+A restful API to sell mobile phones
+
+## Installation
+
+1. Clone the project with `git clone`
+1. Run `composer install`
+1. Create a database and configure the `.env` file with your database credentials
```
Comment (olix21) [inline README.md]: .env.local

File: config/jwt/private.pem [hunk 2]
```diff
@@ -0,0 +1,54 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```
Comment (olix21) [inline config/jwt/private.pem:1]: Do not put it in git

File: src/Controller/ProductController.php [hunk 3]
```diff
@@ -103,120 +111,134 @@ public function getProductsAction(
...
-    public function deleteAction(Product $product, EntityManagerInterface $manager)
+    public function deleteProductAction(Product $product, EntityManagerInterface $manager)
     {
+        $id = $product->getId();
         $manager->remove($product);
         $manager->flush();
-        $view = $this->view($product, Response::HTTP_ACCEPTED);
 
-        return $this->handleView($view);
+        return  $this->view("Product {$id} deleted.", Response::HTTP_OK);
```
Comment (olix21) [inline src/Controller/ProductController.php]: ℹ️ In an api, we usually don't return any "notification" saying that the product was deleted. The HTTP code already says that

**Review by olix21 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nicordev) [reply README.md]: ok

File: config/jwt/private.pem (hunk 2 above)
Comment (nicordev) [reply config/jwt/private.pem:1]: ok

File: src/Controller/ProductController.php (hunk 3 above)
Comment (nicordev) [reply src/Controller/ProductController.php]: ok

**Comment by nicordev** [comment]
Will close #2 #5 #7 #8 #9

**Comment by nicordev** [comment]
Will close #2 #5 #7 #8 #9

**Pull request merged**